src/main/drivers/transponder_ir_ilap.c

```diff
@@ -36,8 +36,7 @@ void transponderIrInitIlap(transponder_t *transponder){
     transponder->vTable             = &ilapTansponderVTable;
     transponder->timer_hz           = TRANSPONDER_TIMER_MHZ_ILAP;
     transponder->timer_carrier_hz   = TRANSPONDER_CARRIER_HZ_ILAP;
-    memset(&(transponder->transponderIrDMABuffer.ilap), 0, TRANSPONDER_DMA_BUFFER_SIZE_ILAP);
-
+    memset(&(transponder->transponderIrDMABuffer.ilap), 0, sizeof(transponder->transponderIrDMABuffer.ilap));
```

Comment:
I think this is unrelated to gcc. Should this go into a separate pull request? Also, is this define now unused and should it be removed?

Reply:
why this change is needed are descriped in the pull request message.
also of coure the marko is still in use. transponderIrDMABuffer uses this to determine the array size

Reply:
Still, it's a change that is unrelated to changing the gcc version, so it should be a separate pull request.